gax/src/main/java/com/google/api/gax/rpc/OperationCallableImpl.java

```diff
@@ -86,11 +86,9 @@
   }
 
   OperationFutureImpl<ResponseT, MetadataT> futureCall(ApiFuture<OperationSnapshot> initialFuture) {
-    RetryingCallable<RequestT, OperationSnapshot> callable =
-        new RetryingCallable<RequestT, OperationSnapshot>(
-            callContextPrototype,
-            new OperationCheckingCallable<RequestT>(longRunningClient, initialFuture),
-            executor);
+    RecheckingCallable<RequestT, OperationSnapshot> callable =
+        new RecheckingCallable<>(
```

Comment:
Please consider addin in `Callables` class the something like the following method:
```java
public static RecheckingCallable<RequestT, OperationSnapshot> rechecking(LongRunningClient longRunningClient, ApiFuture<OperationSnapshot> initialFuture, RetryingExecutor<OperationSnapshot> executor) {
    return new RecheckingCallable<>(callContextPrototype, new OperationCheckingCallable<RequestT>
            (longRunningClient, initialFuture), executor);
}
```
and then calling it here:
```java
RecheckingCallable<> callable = Callables.rechecking(longRunningClient, initialFuture, executor)
```

Reply:
It doesn't quite fit the pattern of Callables currently, which mints new callables using settings classes. I have to think over the design some more, so I'll leave it for another day.